### PR TITLE
[FT:] fix pause tests

### PR DESCRIPTION
**Pull request by dmaish**
### Problem
the pause test misses test to check if the request is a 'POST' request.
-the test_it_works doesn't assert the expected status code and check's status.

### Solution
create a testcase to check if only 'POST' requests are allowed.
create the assert statements to test the expected status code and check's status.

[https://www.pivotaltracker.com/story/show/158729308]

### CODE REVIEW

File: hc/api/tests/test_pause.py [hunk 1]
```diff
@@ -12,16 +12,28 @@ def test_it_works(self):
         r = self.client.post(url, "", content_type="application/json",
```
Comment (thoth-ky) [inline hc/api/tests/test_pause.py]: use meaningful variable names

File: hc/api/tests/test_pause.py [hunk 2]
```diff
@@ -12,16 +12,28 @@ def test_it_works(self):
...
+        
 
     def test_it_validates_ownership(self):
         check = Check(user=self.bob, status="up")
         check.save()
-
+    
         url = "/api/v1/checks/%s/pause" % check.code
-        r = self.client.post(url, "", content_type="application/json",
+        r = self.client.post(url,"", content_type="application/json",
```
Comment (thoth-ky) [inline hc/api/tests/test_pause.py]: space after `,`

**Review by thoth-ky — COMMENTED** [review]
adhere to pep8

**Comment by dmaish** [comment]
@jmutuku95 , full name variables implemented and redundant spaces removed.

**Review by thoth-ky — APPROVED** [review]
Nice

**Review by loicemeyo — APPROVED** [review]

**Review by miritih — APPROVED** [review]
Code works as expected and it follows the required standards.

File: hc/api/tests/test_pause.py [hunk 3]
```diff
@@ -7,21 +7,31 @@ class PauseTestCase(BaseTestCase):
...
+        check.save()
 
-        self.assertEqual(r.status_code, 400)
+        url = "/api/v1/checks/%s/pause" % check.code
+        response = self.client.get(url, "", content_type="application/json",
+                             HTTP_X_API_KEY="abc")
+        self.assertEqual(response.status_code, 405)
 
-        ### Test that it only allows post requests
+
```
Comment (karanjaE) [inline hc/api/tests/test_pause.py:37]: You need an empty line at the end of your files otherwise github complains.

**Review by karanjaE — APPROVED** [review]

**Pull request merged**